templates/security/vulnerabilities/index.html

```diff
@@ -1,20 +1,107 @@
 {% extends "security/base_security.html" %}
 
-{% block title %}Vulnerabilities{% endblock %}
+{% block title %}Vulnerability Knowledge Base{% endblock %}
 
-{% block meta_description %}{% endblock %}
+{% block meta_description %}
+  A library with in-depth information and technical guidance for high-impact vulnerabilities that affect Ubuntu.
+{% endblock %}
 
 {% block meta_copydoc %}
-
+  https://docs.google.com/document/d/1nN_3QGibrmWSi4AAYfoO04q7EnQvHdSgDTaAXvy_4nc/edit?tab=t.0
 {% endblock meta_copydoc %}
 
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
-<div class="section--hero">
-  <div class="row--25-75">
-    <div class="col">
-      <h1>Vulnerabilities knowledge base</h1>
-      <p>Applying security updates in a timely manner is critical to reducing risks, especially with high-impact vulnerabilities. Canonical releases Ubuntu Security Notices whenever a security fix is available for an official Ubuntu package.  When high-profile vulnerabilities are publicly disclosed, the Ubuntu Security Team also provides in-depth technical explanations and mitigation guidance in the form of Vulnerability Knowledge Base articles.</p>
+  <div class="p-section--hero">
+    <div class="row--25-75">
+      <div class="col">
+        <h1 class="p-heading--2">Vulnerability knowledge base</h1>
+        <p>
+          Applying security updates in a timely manner is critical to reducing risks, especially with high-impact vulnerabilities. Canonical releases <a href="/security/notices">Ubuntu Security Notices</a> whenever a security fix is available for an official Ubuntu package.
+        </p>
+        <p>
+          When high-profile vulnerabilities are publicly disclosed, the Ubuntu security team also provides in-depth technical explanations and mitigation guidance in the form of vulnerability knowledge base articles.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--25-75-on-large">
+      <div class="col">
```

Comment:
```suggestion
  <div class="p-section">
    <div class="row--25-75-on-large">
        <hr class="p-rule" />
      <div class="col">
```

Reply:
I prefer not having the `<hr>` be dependent on the `<div class="row">`  for a couple reasons:
1. It is unaffected by any layout changes
2. the parent being `25-75` is misleading, as it actually spans the full width